bitnami/common/templates/_validations.tpl

```diff
@@ -65,18 +65,20 @@ Validate value params:
   - secret - String - Required. Name of the secret where mysql values are stored, e.g: "mysql-passwords-secret"
 */}}
 {{- define "common.validations.values.mariadb.passwords" -}}
-  {{- if and (not .context.Values.mariadb.existingSecret) .context.Values.mariadb.enabled -}}
+  {{- if and (not .context.Values.mariadb.existingSecret) .context.Values.mariadb.enabled .context.Values.mariadb.secret.requirePasswords -}}
```

Comment:
Remove `.context.Values.mariadb.secret.requirePasswords` according with the other comment.